scipy/linalg/basic.py

```diff
@@ -1256,11 +1256,15 @@ def pinv(a, cond=None, rcond=None, return_rank=False, check_finite=True):
     a : (M, N) array_like
         Matrix to be pseudo-inverted.
     cond, rcond : float, optional
-        Cutoff for 'small' singular values in the least-squares solver.
-        Singular values smaller than ``rcond * largest_singular_value``
-        are considered zero.
-        If None, it is set to ``np.finfo(a.dtype).eps``.
-        If `a` is an array of integers, it is set to ``np.finfo('float64').eps``.
+        Cutoff for 'small' singular values; singular values smaller than this
+        value are considered zero. If both are omitted, the default
```

Comment:
I'm still not convinced this is an improvement. The old doc makes it clear that the cutoff provided is relative to the maximum singular value:

> Singular values smaller than ``rcond * largest_singular_value``

The wording of the new one makes it sound like an absolute cutoff (i.e., in each singular value's actual value, not relative to the maximum singular value):

> singular values smaller than this value are considered zero

Reply:
Because it is an absolute cutoff. We specifically use this value passed to lstsq and it overrides it. That was the problem we are fixing.

Reply:
Well if we are to follow all the way to lstsq ok I know what you mean. Let me reword that part.

Reply:
If it were an absolute cutoff (not relative to the max singular value) then the test that I wrote would fail for the scale factor of 1e-20, no? Maybe I am misusing terminology somehow, but it seems like it must be using a "relative" singular value cutoff.

And either way, the docstring in these should match the docstring in `lstsq`, which still uses this relative terminology.

Reply:
No I see your point. It is just that the terminology is really confusing for me since these are quite nonstandard usage.